pcsx2/gui/GlobalCommands.cpp

```diff
@@ -187,6 +198,7 @@ namespace Implementations
 			default: break;
 		}
 
+		OSDlog( wxColor(L"BLUE"), "(GSwindow) Aspect ratio: %s.", WX_STR(arts));
```

Comment:
WX_STR() is a wide character string on Windows - it can't be used because the string will be truncated.